config/environment.js

```diff
@@ -20,6 +20,9 @@ module.exports = function(environment) {
     sentry: {
       environment,
     },
+    apollo: {
```

Comment:
Use an environmental variable instead of hard-coding the URL. And use the official GraphQL endpoint (`/graphql` instead of `/client-api/graphql`).